### PR TITLE
added Manifest Generator utility

### PR DESCRIPTION
**Pull request by milindoke**
*Issue #, if available:*

*Description of changes:*
added Manifest Generator utility

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by rjvgupta — APPROVED** [review]

File: src/.DS_Store
Comment (almann) [inline src/.DS_Store:1]: Please remove this file.

File: src/ManifestGenerator/manifestgen.py [hunk 1]
```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env python3
```
Comment (almann) [inline src/ManifestGenerator/manifestgen.py:1]: I didn't put the appropriate copyright header in the original--that should be required, right?

File: src/ManifestGenerator/manifestgen.py [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+
+import argparse
+import csv
+import io
+import json
+import dataclasses
+import sys
+import typing
+import random
+import boto3
```
Comment (almann) [inline src/ManifestGenerator/manifestgen.py:11]: Note that in the original version, I had a `requirements.txt`, should we have that as well, do we need a `setup.py` or equivalent?

**Review by almann — CHANGES_REQUESTED** [review]

File: src/ManifestGenerator/manifestgen.py (hunk 1 above)
Comment (milindoke) [reply src/ManifestGenerator/manifestgen.py:1]: i have modified your original README.md file and added # License which covers Copyright. so this should be covered.

File: src/ManifestGenerator/manifestgen.py (hunk 2 above)
Comment (milindoke) [reply src/ManifestGenerator/manifestgen.py:11]: the new README.md file i created includes the ## Requirements section. i interpreted this `requirements.txt` to be just a text file and that it was not really required for script execution. 

regarding `setup.py` i do not see it for other utilities on this repo so i am guessing its not required.

File: src/.DS_Store
Comment (milindoke) [reply src/.DS_Store:1]: done

File: src/ManifestGenerator/manifestgen.py (hunk 2 above)
Comment (almann) [reply src/ManifestGenerator/manifestgen.py:11]: Looks like we have it here https://github.com/awslabs/amazon-redshift-utils/blob/master/src/requirements.txt
It should be fine then

**Review by almann — APPROVED** [review]
🚢 LGTM

**Review by saeedma8 — APPROVED** [review]

File: src/.DS_Store
Comment (sgromoll) [inline src/.DS_Store:1]: Please remove this file.  Unnecessary and causing conflicts.

**Review by saeedma8 — APPROVED** [review]

**Review by saeedma8 — APPROVED** [review]

**Pull request closed without merging**